packages/config/src/PrismaConfig.ts

```diff
@@ -15,7 +15,7 @@ type EnvVars = Record<string, string | undefined>
 const sqlMigrationAwareDriverAdapterFactoryShape = <Env extends EnvVars = never>() =>
   Shape.declare(
     (input: any): input is (env: Env) => Promise<SqlMigrationAwareDriverAdapterFactory> => {
-      return input instanceof Function
+      return typeof input === 'function'
```

Comment:
jiti uses the vm module internally to load files. As a result, the execution context in which the parsing occurs differs from the one in which prisma.config.ts is loaded, causing instanceof Function to return false. To avoid this issue, we modified the logic to use typeof for the type check instead.